en/config_mc/mc_slew_rate_type_trajectory.md

```diff
@@ -34,10 +34,10 @@ This parameter is used for position-control in the horizontal direction, where t
 The limit for the rate of change of the velocity setpoint is defined by [MPC_ACC_HOR_MAX](../advanced_config/parameter_reference.md#MPC_ACC_HOR_MAX).
 This parameter should be set larger than any of the other acceleration related parameters in the horizontal direction.
 
-<span id="mpc_acc_hor-and-mpc_dec_hor_slow"></span>
+<a id="mpc_acc_hor-and-mpc_dec_hor_slow"></a>
 #### MPC_ACC_HOR and MPC_DEC_HOR_SLOW
 
-In velocity-control the rate limit for the velocity setpoint is extracted from a linear map from stick input to acceleration limit with maximum [MPC_ACC_HOR](../advanced_config/parameter_reference.md#MPC_ACC_HOR) and minimum [MPC_DEC_HOR_SLOW](../advanced_config/parameter_reference.md#MPC_DEC_HOR_SLOW).
+In velocity-control the rate limit for the velocity setpoint is extracted from a linear map from stick input to acceleration limit with maximum [MPC_ACC_HOR](../advanced_config/parameter_reference.md#MPC_ACC_HOR) and minimum [MPC_DEC_HOR_SLOW](../advanced_config/parameter_reference.md#MPC_DEC_HOR_SLOW). 
```

Comment:
@MaEtUgR `MPC_DEC_HOR_SLOW` appears to have been removed. How should this section (and other mentions in this doc change?)

Reply:
The position mode implementation (flight task) that used this was removed in https://github.com/PX4/PX4-Autopilot/pull/16052
So the entire slew-rate-based trajectory is deprecated.